rosbag2_cpp/include/rosbag2_cpp/message_definitions/local_message_definition_source.hpp

```diff
@@ -75,6 +76,7 @@ class ROSBAG2_CPP_PUBLIC LocalMessageDefinitionSource final
     MSG = 1,
     IDL = 2,
     SRV = 3,
+    ACT = 4,
```

Comment:
`.srv` -> `SRV`, `.action` -> should be `ACTION`?

```suggestion
    ACTION = 4,
```

Reply:
Yeah, that would make sense if, for example, `SRV` is referring to the `/srv/` in service names.